### PR TITLE
encodePacked_16_8_128 v2.

### PR DESCRIPTION
**Pull request by bh2smith**
Addresses issue in #68

### CODE REVIEW

File: test/snapp_utils.js [hunk 1]
```diff
@@ -42,16 +42,13 @@ const uint16 = function(num) {
 
 // returns byte string of hexed-sliced-padded int
 const uint128 = function(num) {
-  assert(num < 2**128)
+  // assert(num < 2**128)
```
Comment (fleupold) [inline test/snapp_utils.js]: Is this intended?

**Review by fleupold — APPROVED** [review]
Is there a way to add a unit test that would have caught this? As in it would have failed before the change but pass now?

File: test/snapp_utils.js (hunk 1 above)
Comment (bh2smith) [reply test/snapp_utils.js]: Nope! Whoops!

File: scripts/claim_withdraw.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ module.exports = async (callback) => {
     for (let i = 0; i < all_withdraws.length; i++) {
       const withdraw = all_withdraws[i]
       if (withdraw.valid) {
-        withdraw_hashes[i] = encodePacked_16_8_128(withdraw.accountId, withdraw.tokenId, withdraw.amount)
+        withdraw_hashes[i] = encodePacked_16_8_128(withdraw.accountId, withdraw.tokenId, parseInt(withdraw.amount))
```
Comment (josojo) [inline scripts/claim_withdraw.js:71]: Do we need this parse, if we have the other parse uint128(c) later on?

**Review by josojo — APPROVED** [review]
Much cleaner than the solutions we discussed :) sweet

**Pull request merged**